.eslintrc

```diff
@@ -7,6 +7,7 @@
   "rules": {
     "flowtype/no-weak-types": 0,
     "flowtype/require-parameter-type": 0,
-    "flowtype/require-return-type": 0
+    "flowtype/require-return-type": 0,
+    "id-length": 0
```

Comment:
Oh right, I was thinking about it from the stand point of what was being "run" and imported during a test, but that's not how the linting works. Sorry, brain fart :P I think I just needed to use you as a rubber duck! :P